ci-operator/config/Intersmash/intersmash/Intersmash-intersmash-main.yaml

```diff
@@ -23,10 +23,6 @@ tests:
       commands: |
         ls -la .ci/openshift-ci/build-root/
         /bin/bash .ci/openshift-ci/build-root/e2e-test.sh
-      credentials:
```

Comment:
Yep, because we don't need to use credentials for pulling images when running tests against community deliverables. Those are available on repos/registries (e.g.: operatorhub.io) which don't require authentication.
That part has been moved to the periodic job definition, which will test against prod deliverables, see https://github.com/openshift/release/pull/43119/files#diff-9875d5acffeb314c124c681a37b4c51c8e86c0c71669eb72f85abb19a2375635R27-R30